Meta/WPT.sh

```diff
@@ -221,17 +221,28 @@ import_wpt()
         INPUT_PATHS[i]="$item"
     done
 
-    TESTS=()
+    RAW_TESTS=()
     while IFS= read -r test_file; do
-        TESTS+=("$test_file")
+        RAW_TESTS+=("$test_file")
     done < <(
         "${ARG0}" list-tests "${INPUT_PATHS[@]}"
     )
-    if [ "${#TESTS[@]}" -eq 0 ]; then
+    if [ "${#RAW_TESTS[@]}" -eq 0 ]; then
         echo "No tests found for the given paths"
         exit 1
     fi
 
+    for i in "${!RAW_TESTS[@]}"; do
+        item="${RAW_TESTS[i]}"
+        item="${item%%\?*}"
+        RAW_TESTS[i]="$item"
+    done
```

Comment:
Wouldn't it be simpler to do `RAW_TESTS+=("${test_file%%\?*}")` in the first loop?

Reply:
No idea why I thought that extra loop was necessary :sweat_smile:. Moved into the first loop.